Xwt.XamMac/Xwt.Mac/TextEntryBackend.cs

```diff
@@ -356,6 +356,29 @@ public CustomCell ()
 
 			}
 
+			protected CustomCell(IntPtr ptr) : base(ptr)
+			{
+			}
+
+			/// <summary>
+			/// Like what happens for the ios designer, AppKit can sometimes clone the native `NSTextFieldCell` using the Copy (NSZone)
+			/// method. We *need* to ensure we can create a new managed wrapper for the cloned native object so we need the IntPtr
+			/// constructor. NOTE: By keeping this override in managed we ensure the new wrapper C# object is created ~immediately,
+			/// which makes it easier to debug issues.
+			/// </summary>
+			/// <returns>The copy.</returns>
+			/// <param name="zone">Zone.</param>
+			public override NSObject Copy(NSZone zone)
+			{
+				// Don't remove this override because the comment on this explains why we need this!
+				var newCell = (CustomCell)base.Copy(zone);
+				newCell.editor = editor;
+				newCell.selChangeObserver = selChangeObserver;
```

Comment:
Do we want to copy an event registration from *a different c# object* to this C# object? When the event is raised the `HandleSelectionDidChange` method will be invoked on the *original* object only, not on the clone.

Reply:
(I ask because it'll be confusing/weird to have a non-null field. it won't really cause a bug though).

Reply:
Doesn't copying also copy everything else?